package.json

```diff
@@ -93,6 +87,13 @@
     "graphql-request": "^3.4.0",
     "inter-ui": "^3.13.1",
     "jest-styled-components": "^7.0.5",
+    "legacy-web3-react/core": "npm:@web3-react/core",
```

Comment:
These should retain the same name so that the current interface can continue to use them unchanged. The new version should be renamed.